packager/media/formats/mp2t/ts_segmenter.h

```diff
@@ -69,6 +69,10 @@ class TsSegmenter {
 
   /// Only for testing.
   void SetTsWriterFileOpenedForTesting(bool value);
+  
+  /// @param pts value of segment_start_timestamp_.
+  void SetSegmentStartTimestamp(int pts) { segment_start_timestamp_ = pts;};
+  int GetSegmentStartTimestamp() {return segment_start_timestamp_; };
```

Comment:
Two questions:

(1) Does the two functions need to be public?
(2) If not, do we need the two functions, i.e. we could set and get |segment_start_timestamp_| in the code directly.

Reply:
Yes we can. Will remove the extra functions.